### PR TITLE
Feature/cybersource creditcard authorize

### PR DESCRIPTION
**Pull request by harshrsys**
## PR Type

[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no API changes)
[ ] Build-related changes
[ ] CI-related changes
[ ] Documentation content changes
[ ] Application / infrastructure changes
[ ] Other: 

## What Is the New Behavior?
Implementation for Cybersource credit card payment method.
## Does this PR Introduce a Breaking Change?

[ ] Yes
[X] No

## Other Information
Requires CyberSource Payment Connector 1.0.0 - still unreleased

### CODE REVIEW

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  font-style: normal;
+  font-weight: 400;
+  font-variant: normal;
+  font-size-adjust: none;
+  font-stretch: 100%;
+  line-height: 20px;
+  color: rgb(85, 85, 85);
+  background-color: rgb(255, 255, 255);
+  background-image: none;
+  border-top-color: rgb(204, 204, 204);
```
Comment (SGrueber) [inline src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:25]: border:  ...

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.flex-microform {
```
Comment (SGrueber) [inline src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:1]: pls reduce the number of styles here, are they all really necessary?

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  border-bottom-left-radius: 2px;
+  box-shadow: rgba(0, 0, 0, 0.075) 0 1px 1px 0 inset;
+  transition-delay: 0s, 0s;
+  transition-timing-function: ease-in-out, ease-in-out;
+  transition-duration: 0.15s, 0.15s;
+  transition-property: border-color, box-shadow;
+}
+
+/* different styling for a specifc container */
+
```
Comment (SGrueber) [inline src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:49]: remove these comments

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+/* background: purple; */
+
+/* } */
+
+/* .flex-microform-focused {} */
+
+/* .flex-microform-valid {} */
+
+.flex-microform-valid.flex-microform-focused {
+  border-top-color: rgb(32, 181, 181);
```
Comment (SGrueber) [inline src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:61]: border

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  border-right-color: rgb(32, 181, 181);
+  border-bottom-color: rgb(32, 181, 181);
+  border-left-color: rgb(32, 181, 181);
+  box-shadow: 0 0 8px 0 rgba(102, 175, 233, 0.6);
+}
+
+/* .flex-microform-autocomplete {} */
```
Comment (SGrueber) [inline src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:68]: remove

**Review by SGrueber — CHANGES_REQUESTED** [review]

File: src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss (hunk 2 above)
Comment (harshrsys) [reply src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:1]: Kindly provide more details on this.

Comment (SGrueber) [reply src/app/pages/checkout-payment/payment-cybersource-creditcard/payment-cybersource-creditcard.scss:1]: It seemed to me, some style definition are not necessary here, e.g. is it really necessary to define the font-stretch, e.g.? So I wanted you to check this. If you think, we need all these styles leave it as it is now.

**Pull request merged**